File: pgm/ABC_Visual_Basic/ABC_144_A.vb
' Imports System.Collections.Generic
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1)
        Console.WriteLine(If(A <= 9 AndAlso B <= 9, A * B, -1))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_144_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int A, B;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        Console.WriteLine(A <= 9 && B <= 9 ? A * B : -1);
        return;
    }
}